apps/src/lib/ui/InfoHelpTip.jsx

```diff
@@ -2,17 +2,20 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import ReactTooltip from 'react-tooltip';
+import {BodyTwoText} from '@cdo/apps/componentLibrary/typography';
 
 export default function InfoHelpTip({id, content}) {
   return (
-    <div>
+    <span style={{marginLeft: '12px'}}>
```

Comment:
I know this is nit-picky, but from a code style perspective, would we rather have all of this CSS in the `styles` variable at the end of of file?  Since this was only one style change, I felt like it was overkill to add it to the `styles` variable - it seemed to make it less readable to me, but I'd love to know the hive's thoughts here.